src/lib/tls/tls_algos.h

```diff
@@ -198,7 +203,7 @@ class BOTAN_PUBLIC_API(3, 2) Group_Params final {
                 m_code == Group_Params_Code::FFDHE_8192;
       }
 
-      constexpr bool is_pure_kyber() const {
+      constexpr bool is_pure_kyber_r3() const {
```

Comment:
This is a public API. I think we'll have to go with deprecation of `is_pure_kyber()` here.